app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class TargetsController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def create
+        target = current_api_v1_user.targets.create!(target_params)
+
+        render json: target, status: :created
```

Comment:
NIT:
Since we are adding jbuilder in this PR, how about we create a view that returns this data?